src/rust/engine/process_execution/src/remote.rs

```diff
@@ -1120,22 +1193,36 @@ mod tests {
       env: BTreeMap::new(),
       input_files: fs::EMPTY_DIGEST,
       output_files: BTreeSet::new(),
+      timeout: Duration::from_millis(5000),
+      description: "echo a foo".to_string(),
     }
   }
 
-  fn make_incomplete_operation(operation_name: &str) -> bazel_protos::operations::Operation {
+  fn make_incomplete_operation(
+    operation_name: &str,
+  ) -> (bazel_protos::operations::Operation, Option<Duration>) {
```

Comment:
The `Option<Duration>` on this function (and the other hard-coded-to-`None` returning functions) probably deserves a comment explaining that it's so that callers can just the output of these calls compatibly in a `Vec` with calls to `make_delayed_incomplete_operation`

Reply:
Makes sense to me.